### ISSUE TITLE
Improve freeze Snow

### ISSUE THREAD

**Issue by C0D3D3V** [ddnet/ddnet#5581]
Requested by cheeser0613 on discord (and a lot of other also did smaller requests)

"The stars and ninja particles were not spawned on the tee:"
![grafik](https://user-images.githubusercontent.com/14315968/177598260-b5266072-ce8a-40dc-b2f4-56fcb9d11e14.png)
"The snow flakes spawn on the tee and a lot slower" (to be precise 10 times slower, from 100Hz down to 10Hz)
![grafik](https://user-images.githubusercontent.com/14315968/177598366-d7cb5f26-a0fd-4699-b3c0-6599a8505a3a.png)

```
its not about how beauty the effect i want sometimes
im more prefer on what advantage it have
but i cant 100% say this is bad, because it might also just me that not used to this new effect
so if we're able to decide which effect we can use in future then i guess it will be good
```

-> Face of tees in freeze are full of snow
-> Snow does not produce as good a track as the Ninja sword particles.

```
it doesnt spawn "on" the tee, the visual on tee is more clear than new one
```
![grafik](https://user-images.githubusercontent.com/14315968/177598726-be94258d-0b18-4be8-901e-4bf56fa492df.png)
![grafik](https://user-images.githubusercontent.com/14315968/177598739-13fde9df-88d9-420a-97bd-281bcd5bd57b.png)

```
maybe i require too much, but is it possible to raise the spawn rate of the effect?
```
![grafik](https://user-images.githubusercontent.com/14315968/177598824-d8afe254-0b0b-4371-806e-0ce8b46050bd.png)

Example with higher spawn rate:
https://cdn.discordapp.com/attachments/892695271723638815/993855123946229870/dddddddddd.demo.mp4
![grafik](https://user-images.githubusercontent.com/14315968/177598873-7f206908-e6de-4a88-8dc8-d3e584f515f9.png)

**Comment by Learath2** [comment]
I tested with making the "spawning box" 32 by 10 and moving it to the bottom of the tee. That combined with making the snow about 3hz looked better. Maybe start there if you want to mess around with it.

I heard from some people that some of the flakes are also too big so maybe decrease the max size a little?

**Comment by C0D3D3V** [comment]
Yes I also thought about that, since this is more complicated then just changing the spwan position, I will first try to play with the position the snowflakes spwan. The ninja sword particles always spwaned to the left or right of you. We can do something similar and spwan the snowflakes if you sit only in the lower part of you, (or maybe also on the sides, but not on the face) and if you are not still using the movement direction vector to set the spwan area around the tee.
